retroauth/src/main/java/com/andretietz/retroauth/OwnerStorage.kt

```diff
@@ -25,17 +25,14 @@ interface OwnerStorage<in OWNER_TYPE : Any, OWNER : Any, in CREDENTIAL_TYPE : An
 
   /**
    * Creates an [OWNER] of a specific [ownerType] for a specific [credentialType]. So open a login and let the user
-   * login. If the User cancels the login an [AuthenticationCanceledException] should be thrown.
+   * login. If the User cancels, [Callback.onError] should be called with an [AuthenticationCanceledException].
```

Comment:
```suggestion
   * login. If the User cancels, [Callback.onError] should be called with an [AuthenticationCanceledException] or if
   * the [Future.get] is used, it'll throw that Exception
```

FYI: This case needs also to be covered:
```kotlin
    buttonLogin.setOnClickListener {
      Single.fromCallable {
        ownerManager.createOwner(provider.ownerType, provider.credentialType).get()
      }.subscribeOn(Schedulers.io())
        .observeOn(AndroidSchedulers.mainThread())
        .subscribe {
          account: Account?, error: Throwable? ->
          if(account != null) {
            Timber.d("Logged in: $account")
          }
          if(error != null) {
            showError(error)
          }
        }
    }
```